### PR TITLE
feat(elasticsearch): improve mappings - EUBFR-88

### PR DESCRIPTION
**Pull request by kalinchernev**
Todos after feedback from Rainer:
- Evaluate https://www.elastic.co/guide/en/elasticsearch/reference/current/range.html for dates

Deployed stage `chernka30`, so far do not yield errors on portal side => works.
I'm awaiting some additional feedback from Rainer if there are some other tweaks necessary before we merge as `test` stage is frozen for the time being.

### CODE REVIEW

**Comment by kalinchernev** [comment]
Removing the `wip` label as Rainer hasn't reported any issues with current implementation.
Geospatial follow-up on EUBFR-97, EUBFR-98 and also EUBFR-99 is to help Rainer with specific search queries as in current portal.

File: services/value-store/projects/src/events/onObjectCreated.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ import split2 from 'split2';
 
 import deleteProjects from '../lib/deleteProjects';
 import SaveStream from '../lib/SaveStream';
+// elasticsearch mapping https://goo.gl/MEGiA6
```
Comment (yhuard) [inline services/value-store/projects/src/events/onObjectCreated.js]: Please don't use Google's URL shortener... please...

File: services/value-store/projects/src/mappings/project.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      producer_id: {
+        type: 'keyword',
+      },
+      programme_name: {
+        type: 'keyword',
+      },
+      project_locations: {
+        properties: {
+          location: {
+            type: 'geo_point',
```
Comment (yhuard) [inline services/value-store/projects/src/mappings/project.js]: Should be `geo_shape` https://www.elastic.co/guide/en/elasticsearch/reference/current/geo-shape.html

File: services/value-store/projects/src/mappings/project.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            ignore_malformed: true,
```
Comment (yhuard) [inline services/value-store/projects/src/mappings/project.js]: Is there a specific reason for `ignore_malformed: true`?

File: services/value-store/projects/src/events/onObjectCreated.js (hunk 1 above)
Comment (kalinchernev) [reply services/value-store/projects/src/events/onObjectCreated.js]: I didn't fine one for the EC and links are sometimes too long and ugly

File: services/value-store/projects/src/mappings/project.js (hunk 3 above)
Comment (kalinchernev) [reply services/value-store/projects/src/mappings/project.js]: This one I added before we make latitude and longitude 0 if not available. If a value is not 0, but `null` as in most defaults in our case, indexing fails and stops the process.

File: services/value-store/projects/src/mappings/project.js (hunk 2 above)
Comment (kalinchernev) [reply services/value-store/projects/src/mappings/project.js]: I'd suggest this change to happen in EUBFR-98 where we ensure the transform is also updated?

**Pull request merged**